etcdctl/README.md

```diff
@@ -837,28 +837,73 @@ Prints a line of JSON encoding each endpoint URL and KV history hash.
 Get the hash for the default endpoint:
 
 ```bash
-./etcdctl endpoint hashkv
-# 127.0.0.1:2379, 1084519789
+./etcdctl endpoint hashkv --cluster
+http://127.0.0.1:2379, 2064120424, 13
+http://127.0.0.1:22379, 2064120424, 13
+http://127.0.0.1:32379, 2064120424, 13
 ```
 
 Get the status for the default endpoint as JSON:
 
```

Comment:
Could you, please explain here what's the relationship between `hash_revision` and `revision`.
Why hash_revision can be stale and inconsistent with other members.

Reply:
`hash_revision` is the revision up to which the hash is calculated. `revision` included in the response header is the latest revision the member has. Usually `revision` may be bigger than `hash_revision` when the cluster is under high load. 

`./etcdctl endpoint hashkv --cluster` just get each member's hash one by one, so the hash_revision might be inconsistent. Previously I was thinking to pre-fetch the revision, and then get hash using the same revision. But etcdctl is a CLI tool, and I expect users retry the command with a revision when they see inconsistent hash_revision